build/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:bionic
+
+RUN apt update && \
+    apt install -y git  \
+    openjdk-11-jdk-headless \
```

Comment:
How is the redis-server dependency handled? Will the redis server run as host daemon process that will be used by the service running inside the container?